node/src/components/consensus/era_supervisor.rs

```diff
@@ -1300,8 +1296,30 @@ async fn execute_finalized_block<REv>(
             return;
         }
     }
+
+    // Get the deploy hashes for the finalized block.
+    let transaction_hashes = finalized_block.transfer_hashes().to_owned();
+
+    // Get all deploys in order they appear in the finalized block.
+    let mut transactions: Vec<Deploy> = Vec::with_capacity(transaction_hashes.len());
```

Comment:
```suggestion
    // Get all transfers in order they appear in the finalized block.
    let mut transfers: Vec<Deploy> = Vec::with_capacity(transfer_hashes.len());
```

Reply:
Fixed in 8e89b9c98236e80661953dce42d8328fd93a994f